src/wmnds/patterns/buy-a-ticket/_buy-a-ticket.njk

```diff
@@ -0,0 +1,112 @@
+{% macro wmndsBuyATicket() %}
+{# Imports #}
+{% from "wmnds/components/button/_button.njk" import wmndsButton %}
+{% from "wmnds/components/form-elements/dropdown/_dropdown.njk" import wmndsDropdown %}
+
+<div class="wmnds-buy-a-ticket">
+    <h2 class="wmnds-buy-a-ticket__title">Buy a ticket</h2>
+    {# Mode select #}
+    <div class="wmnds-m-b-md wmnds-p-b-xsm">
+        <label for="" class="wmnds-buy-a-ticket__label">Which modes of transport will you use?</label>
+        <div class="wmnds-grid wmnds-grid--spacing-3-sm wmnds-grid--spacing-sm-3-md">
+            <div class="wmnds-col-1-3">
+                {{ 
+                    wmndsButton({
+                        iconLeft: "modes-isolated-bus",
+                        contentText: "Bus",
+                        type: "mode",
+                        classes: "wmnds-btn--block wmnds-is--active"
+                    })
+                }}
+            </div>
+            <div class="wmnds-col-1-3">
+                {{ 
+                    wmndsButton({
+                        iconLeft: "modes-isolated-rail",
+                        contentText: "Train",
+                        type: "mode",
+                        classes: "wmnds-btn--block"
+                    })
+                }}
+            </div>
+            <div class="wmnds-col-1-3">
+                {{ 
+                    wmndsButton({
+                        iconLeft: "modes-isolated-metro",
+                        contentText: "Tram",
+                        type: "mode",
+                        classes: "wmnds-btn--block"
+                    })
+                }}
+            </div>
+        </div>
+    </div>
+    {# Traveller dropdown #}
+    {{
+        wmndsDropdown({
+            id: 'buyATicketTraveller',
```

Comment:
If wmndsDropdown accepts name for the input could you add **name: "PassengerType",**, please!